.github/workflows/build.yaml

```diff
@@ -17,14 +17,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          lfs: true
       - uses: gradle/wrapper-validation-action@v1
       - uses: actions/setup-java@v3.5.1
         with:
           distribution: 'zulu'
           java-version: 18
       - uses: gradle/gradle-build-action@v2
 
-      - run: ./gradlew build dokkaHtmlMultiModule --parallel
+      - run: ./gradlew build dokkaHtmlMultiModule verifyPaparazziDebug --parallel
```

Comment:
Oof the non-validating default behavior strikes again.

Reply:
Yep, it caught me by surprise since Paparazzi still runs as part of `test`.